dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -54,7 +54,7 @@
   # Hopefully fixes many of the issues with font wiggle due to lazily loading
   # alternative fonts for symbols and localized glyphs.
   wait_until do
-    @browser.execute_script('return !!(await document.fonts.ready)') == true
+    @browser.execute_script('return document.fonts.ready.then(() => true)') == true
```

Comment:
await can't be used outside of an async function's scope, so converted to a promise chain. also, just found out the promise resolved from `document.fonts.ready` returns `undefined`, not a boolean 😅